ziti-tunnel/docker/docker-entrypoint.sh

```diff
@@ -49,6 +49,12 @@ if [ ! -f "${json}" ]; then
     ziti-tunnel enroll --jwt "${jwt}" --out "${json}"
 fi
 
+# TODO: solve for ARMv8 aka arm64
```

Comment:
I'm not sure why the TODO is here. What is the solution that differs from this workaround? BTW this is not ARM specific.

Reply:
I must have removed that line at some point because I don't see it when I look at the source branch.